src/Core/Components/Camera/CameraEffects.h

```diff
@@ -52,6 +52,11 @@ namespace MxEngine
 
         bool enableFXAA = false;
         uint8_t bloomIterations = 3;
+
+        bool enableGodRay = false;
```

Comment:
I guess it should be a separate component (like SSAO/SSGI/SSR)
1) now its a bit weird that its possible to tweak  god ray params when god ray is off (I checked, but seems that now we do not support conditional display of fields without creating custom editors which is better to avoid)
2) camera effects component is mostly for effects which are enabled by default. I do not think god rays should be enabled by default => better to create an optional component